builders/container-apps-internal-registry-demo/startup-script.sh

```diff
@@ -140,10 +160,18 @@ echo "===== Executing the extend phase ====="
   -log-level debug \
   -app $CNB_APP_DIR
 
+lifecycleExitCode=$?
+if [ $lifecycleExitCode -ne 0 ]; then
+    echo "----- Build failed -----"
+    echo "Build process failed with exit code '$lifecycleExitCode'. Exiting..."
```

Comment:
Please consider:
* Replacing "Build process failed" by "Cloud Build failed". In the CLI we're pushing the "Cloud Build" wording, so it would make sense to be consistent.
* Please consider removing the "Exiting..." part. I don't feel strongly so feel free to keep it if you do, but I don't think it adds much.

Reply:
updated this to be a single line that still has the `-----` for better visibility and distinction from the rest of the logs

Reply:
OK sounds good, thanks!